static/app/components/contextPickerModal.tsx

```diff
@@ -401,16 +404,7 @@ class ContextPickerModal extends Component<Props> {
   }
 }
 
-type ContainerProps = Omit<
-  Props,
-  | 'projects'
-  | 'loading'
-  | 'organizations'
-  | 'organization'
-  | 'onSelectOrganization'
-  | 'integrationConfigs'
-> & {
-  allowAllProjectsSelection?: boolean;
```

Comment:
This `allowAllProjectsSelection` was written twice, this instance 'overrode' the one above.

Now we have some shared stuff, and then specifics for the Container and the Modal itself.